### PR TITLE
Install script improvements

### PR DESCRIPTION
**Pull request by tfalders**
Ports changes made in https://github.com/ROCmSoftwarePlatform/hipSOLVER/pull/67 to hipBLAS, including adding the cmake-arg parameter and allowing rocblas-path and rocsolver-path to be specified as relative paths.

### CODE REVIEW

**Review by rkamd — APPROVED** [review]
LGTM

File: install.sh [hunk 1]
```diff
@@ -579,19 +597,19 @@ pushd .
           echo "Code coverage is disabled in Release mode, to enable code coverage select either Debug mode (-g | --debug) or RelWithDebInfo mode (-k | --relwithdebinfo); aborting";
           exit 1
       fi
-      cmake_common_options="${cmake_common_options} -DBUILD_CODE_COVERAGE=ON"
+      cmake_common_options+=("-DBUILD_CODE_COVERAGE=ON")
   fi
 
   # Build library
   if [[ "${build_relocatable}" == true ]]; then
```
Comment (daineAMD) [inline install.sh:604]: Just for my sake, what's the purpose of this change?

**Review by daineAMD — APPROVED** [review]
